vignettes/data-transform-as-shiny-module.Rmd

```diff
@@ -1,26 +1,30 @@
 ---
-title: "Data Transform as shiny Module"
+title: "Data Transformations as shiny Module"
 author: "NEST CoreDev"
 output: 
   rmarkdown::html_vignette:
     toc: true
 vignette: >
-  %\VignetteIndexEntry{Data Transform as shiny Module}
+  %\VignetteIndexEntry{Data Transformations as shiny Module}
   %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
 ---
 
 ## Introduction
 
-When data transformations need to occur inside the teal app, they typically happen within the teal module development by the teal module developer. This results in the transformation code being tightly coupled with the teal module. Any changes to the data transformation logic require updating the teal module, which can impact the teal app development until the module is updated.
-With the new `teal_transform_module()` function, it is possible to provide a more dynamic data transformation capability that can be customized by the teal app developer, not just the teal module developer.
-The `teal_transform_module()` function is essentially a Shiny module that takes `ui` and `server` arguments and can be provided to the `transformer` argument in a `module()`. When provided, teal will execute the data transformation logic for the specified module when it is loaded. 
-In addition, this approach makes it easier for teal app developers to handle additional data transformations while using the teal modules R packages like `teal.modules.general` and `teal.modules.clinical`.
-Let’s explore this new way to manage custom data transformations in teal apps.
+`teal` ver `0.15.x` (TO BE UDPATED) introduced new argument in `teal::module` called `transformers`. 
+This argument allows to pass a `list` of `teal_data_module` objects that are created either with `teal_transform_module()` or `teal_data_module()` functions.
+
+The main benefit of `teal_transform_module()` is the ability to transform data before passing the 
+data into the module. This feature allows to extend the regular behavior of existing modules by specifying custom data operations on data inside this module.
+
+`teal_transform_module()` is a Shiny module that takes `ui` and `server` arguments. When provided, `teal` will execute data transformations for the specified module when it is loaded and each time the data is changed. `server` extend the logic behind data manipulations, where `ui` extends filter panel with new UI elements that orchestrate the transformer inputs.
+
+This vignette presents the way on how to manage custom data transformations in `teal` apps.
 
 ## Creating your first custom data transformation module
 
-We initialize a simple teal app where we pass `iris` and `mtcars` as the input datasets.
+We initialize a simple `teal` app where we pass `iris` and `mtcars` as the input datasets.
 
 ```{r, message = FALSE, warning = FALSE}
 library(teal)
```

Comment:
Since the `teal` library is already loaded in an earlier section, there's no need to repeat `library(teal)` in this chunk or in any subsequent chunks. WDYT?

Reply:
Thanks, removed instances of redundant library calls.